submissions/yaripey/dom-task/js/main.js

```diff
@@ -45,33 +45,23 @@ const texts = {
 document.addEventListener('DOMContentLoaded', () => {
   const toggler = document.querySelector('.toggle')
 
-  // Function for changing text in main
   const changeText = (target) => {
     const currentMain = document.querySelector('main');
 
-    if (currentMain.classList[0] !== target.id) { // if this page isn't open already
-
-      const newText = texts[target.id].text.map(paragraph => `<p>${[paragraph]}</p>`)
-      currentMain.innerHTML = `
+    const newText = texts[target.id].text.map(paragraph => `<p>${[paragraph]}</p>`)
+    currentMain.innerHTML = `
           <img class="picture" src="${texts[target.id].img}">
           ${newText.join('\n')}
       `
+    const currentButton = document.querySelector('.focused')
+    currentButton.classList.toggle('focused')
+    target.classList.toggle('focused')
 
-      // Changing styles of the corresponding button
-      const currentButton = document.querySelector('.focused')
-      currentButton.classList.toggle('focused')
-      target.classList.toggle('focused')
-
-      // For mobile, closing the menu after button tap
-      if (document.querySelector('aside').classList.contains('toggled')) {
-        document.querySelector('aside').classList.toggle('toggled')
-      }
-    }
+    document.querySelector('aside').classList.remove('toggled')
```

Comment:
why did you remove this logic? If you use the same selector several times, it is better to put it in a separate variable.I meant it.

Reply:
I didn't remove the logic. `.remove()` does almost the same as what my logic did, so there's no point in this `if` statement. But I still moved the selector in the constant, since I indeed use it several times in code, thanks for pointing that out.